dataactcore/local_secrets_example.yml

```diff
@@ -23,34 +23,24 @@ broker:
     use_aws: false
 
     # If using AWS, set your region here
-    aws_region: us-east-1
+    aws_region: aws-region
 
     # Add your AWS Key to use for sending SES emails
-    aws_access_key_id: ACCESSKEYID123
-    aws_secret_access_key: test123abc/123testsecretkeY
+    aws_access_key_id: keyId
+    aws_secret_access_key: accessKey
 
     # Name of AWS S3 bucket for uploaded broker files. Ignored if use_aws
     # is false. NOTE: the dummy value below MUST be changed to the correct
     # value if use_aws is true.
-    aws_bucket: sample-aws-bucket-name
-    
-    # S3 filenames for SF-133 file, only required if planning to load SF-133 table
-    sf_133_bucket: sample-sf133-bucket-name
-
-    # Region for pulling CFDA file, only required if planning to load CFDA file
-    cfda_region: us-east-1
-
-    # Bucket for pulling CFDA file, only required if planning to load CFDA file
-    cfda_bucket: sample-cfda-bucket-name
+    aws_bucket: s3-bucket
 
-    # File path, name for CFDA pile relative to the bucket and formatted to the date provided.
-    # Only required if planning to load CFDA file
-    cfda_file_path: sample-cfda-file-path
+    # S3 filenames for SF-133 file, only required if planning to load SF-133 table
+    sf_133_bucket: s3-bucket
 
     # Name of the AWS role you're using to upload broker files. Ignored if
     # use_aws is false. NOTE: the dummy value below MUST be changed to the
     # correct value if use_aws is true.
-    aws_role: arn:aws:iam::123456789012:role/roleName
+    aws_role: redacted
```

Comment:
aws_role is both here and in the sample config. Here it's redacted, there it isn't. We should pick one and stick to it.